python/grass/gunittest/loader.py

```diff
@@ -117,6 +118,7 @@ def discover_modules(
             full = os.path.join(root, testsuite_dir)
 
             all_files = os.listdir(full)
+            files = [f for f in all_files]
```

Comment:
But also, `all_files` is not really used afterwards, so perhaps `files` is all we need:

```suggestion
            files = os.listdir(full)
```

Reply:
Applied all three filters